libclc/CMakeLists.txt

```
@@ -244,6 +244,17 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
 		endif()
 		message( "    DEVICE: ${d} ( ${${d}_aliases} )" )
 
+		# FIXME: this is a hack, remove once we can use sycldevice in the triple
+		#        without changing the language
+    # see issue: https://github.com/intel/llvm/issues/1814
```

```suggestion
		#        without changing the language
    # see issue: https://github.com/intel/llvm/issues/1814
```
Looks like a mix of tabs and spaces.

---

and of course I forgot this file uses tabs ...

It is fixed